app/src/main/java/kernbeisser/CustomComponents/ControllerButton.java

```diff
@@ -104,8 +124,31 @@ public ControllerButton withConfirmMessage(String confirmMessage) {
 
   public static ControllerButton empty() {
     return new ControllerButton(
-        () -> new ComponentController(new JPanel()),
-        ComponentController.class,
-        Controller::openTab);
+        () -> new ComponentController(new JPanel()), ComponentController.class);
+  }
+
+  private static boolean assumeControllerAccess(PermissionKey[] requiredKeys) {
+    PermissionSet userPermissions = new PermissionSet();
+    Access.runWithAccessManager(
+        AccessManager.ACCESS_GRANTED,
+        () -> userPermissions.addAll(LogInModel.getLoggedIn().getPermissionSet()));
+    return requiredKeys.length == 0 || userPermissions.hasPermissions(requiredKeys);
+  }
+
+  private static <C> boolean checkControllerAccess(Class<C> clazz) {
+    PermissionKey[] keys = {};
```

Comment:
Was passiert in solchen Fällen?
https://github.com/heintzl/Kernbeisser-Gradle/blob/76e65136952d98433086812ee57f7a9958102667/app/src/main/java/kernbeisser/Windows/Menu/MenuController.java#L59-L67)
  -> Da wird die Action-Permission, je nach benutzung des Controllers  im MenuController definiert.